### PR TITLE
License Cleanup

### PR DESCRIPTION
**Pull request by gonsie**
I've cleaned up the license, notice, and copyright information.

1. The license file is now just the BSD-3 clause
2. The additional "notice" information has been moved to a NOTICE file
3. The copyright statements were added to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,5 +97,22 @@ it should be provided in the common library.
 ## Contributors
 We welcome contributions to the project.  For details on how to help, see our [Contributor Guide](.github/CONTRIBUTORS.md)
 
+### Copyrights
+
+Copyright (c) 2013-2015, Lawrence Livermore National Security, LLC.
+  Produced at the Lawrence Livermore National Laboratory
+  Written by Adam Moody <moody20@llnl.gov>.
```
Comment (adammoody) [inline README.md]: Let's kill the Adam Moody line if we can.

File: LICENSE [hunk 2]
```diff
@@ -1,85 +1,29 @@
...
-Copyright (c) 2006-2007,2011-2015, Los Alamos National Security, LLC.
-  (LA-CC-06-077, LA-CC-10-066, LA-CC-14-046)
-
-Copyright (2013-2015) UT-Battelle, LLC under Contract No.
-DE-AC05-00OR22725 with the Department of Energy.
-
-Copyright (c) 2015, DataDirect Networks, Inc.
+BSD 3-Clause License
 
+Copyright (c) 2018, Lawrence Livermore National Security, LLC
```
Comment (adammoody) [inline LICENSE]: Can we list LANL, ORNL, and DDN here, too?

**Comment by gonsie** [comment]
1257 deletions 😄

I left the `src/dcp1/COPYING` file, since many files in that directory refer to it.

This should be ready to merge.

**Comment by adammoody** [comment]
Thanks!  Good call on the COPYING.  That was the original copyright for dcp, so we should leave that in place.

**Pull request merged**